src/main/java/seedu/eduke8/common/DisplayableList.java

```diff
@@ -0,0 +1,15 @@
+package seedu.eduke8.common;
+
+import java.util.ArrayList;
+
+public interface DisplayableList {
+    ArrayList<Displayable> getInnerList();
+
+    void add(Displayable displayable);
+
+    void delete(int index);
+
+    Displayable find(String description);
+
+    int getCount();
+}
```

Comment:
The DisplayableList class seems a bit redundant at the moment as its functions are very similar to an actual ArrayList. We should look into developing this class more to better manipulate the lists in our program.